Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD nakadi /nakadi
 RUN find ./nakadi | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 ADD run.sh /nakadi/run.sh
+ADD scm-source.json /scm-source.json
```

Comment:
This file is usually considered part of the build artifacts, e.g. not versioned.